### DISCUSSION TITLE
Image scroll by mouse dragging

### DISCUSSION THREAD

**Discussion by rageworx** [fire-eggs/YAIV#26] [Ideas]
How about like this ? ( MRawViwer )

![record](https://user-images.githubusercontent.com/6948225/119447505-f59fb680-bd6a-11eb-925d-6107d2e4ad22.gif)

**Comment by fire-eggs** [comment]
Certainly on my to-do list. BTW, I love the "viewport hint" display!

**Reply by fire-eggs** [reply]
Added! It's remembered across invocations as well.

Also, the preferences file has moved from "iglite.com" to "yaiv.org".

**Reply by rageworx** [reply]
> "yaiv.org".

Now you can make a domain name 'https://yaiv.org' to like to https://github.com/fire-eggs !
I need to change my MIOHDR identity name 'rageworx.com' to 'rageworx.info' ... because .com domain too expensive *sigh*

Or get a raspberrypi3b+, and build own server as like me 😆 ( my https://rageworx.info runs on rpi3b+ as well )

**Reply by fire-eggs** [reply]
>Or get a raspberrypi3b+, and build own server as like me 

Neat! I should dust off one of my RPis and try it!
